src/main/webapp/i18n/de/programmingExercise.json

```
@@ -41,6 +41,7 @@
             "buildplans": "Build Pläne",
             "publishBuildPlanUrl": "Build Plan veröffentlichen",
             "allowOnlineEditor": "Online Editor erlauben",
+            "noLocalSetup": "Kein Lokaler Setup",
```

I like this one more, but I don't care that much ;-)
```suggestion
            "noLocalSetup": "Kein lokales Setup",
```

---

I adapted the translation